src/Missings.jl

```diff
@@ -286,6 +297,7 @@ Base.eltype(itr::EachReplaceMissing) = Missings.T(eltype(itr.x))
     (v isa Missing ? itr.replacement : v, s)
 end
 
+@static if !isdefined(Base, :skipmissing)
```

Comment:
Why `@static` here and not in other places? I'm not sure whether it's needed.

Reply:
It's not necessary at the top level (though I think you can use it to get around deprecations from the parser) but it doesn't hurt.